cmd/test/main.go

```diff
@@ -515,3 +519,22 @@ func wereViolationsFound(validationManager *ValidationManager, results *evaluati
 		return false
 	}
 }
+
+func saveDefaultRulesAsFile(ctx *TestCommandContext, preRunPolicies string) {
+	if preRunPolicies == "" {
+		return
+	}
+
+	homeDir, err := os.UserHomeDir()
+	if err != nil {
+		return
+	}
+
+	defaultRulesFilePath := filepath.Join(homeDir, ".datree", "defaultRules.yaml")
+
+	writeErr := ioutil.WriteFile(defaultRulesFilePath, []byte(preRunPolicies), 0644)
+
+	if writeErr != nil {
+		fmt.Printf("Failed to write default rules to file %s: %s", defaultRulesFilePath, writeErr)
```

Comment:
what are the consequences of printing an error in this case? what will the user experience? are we sure we want that experience?